tests/test.helpers.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const checkIsCorrectlyAligned = (cellsArray, alignment, cursor = null) => {
+  if (cellsArray.length !== 4) {
+    throw new Error('Array must have exactly 4 elements');
+  }
+
+  switch (alignment) {
+    case 'start': {
+      let isZeroFoundStart = false;
+
+      for (const cell of cellsArray) {
+        if (cell === 0) {
+          isZeroFoundStart = true;
+        } else if (isZeroFoundStart) {
+          return false;
+        }
+
+        if (cursor !== null && cell === cursor) {
+          break;
+        }
+      }
+
+      return !isZeroFoundStart || cursor !== null;
+    }
+
+    case 'end': {
+      let isZeroFoundEnd = false;
+
+      for (let i = cellsArray.length - 1; i >= 0; i--) {
+        const cell = cellsArray[i];
+
+        if (cell === 0) {
+          isZeroFoundEnd = true;
+        } else if (isZeroFoundEnd) {
+          return false;
+        }
+
+        if (cursor !== null && cell === cursor) {
+          break;
+        }
+      }
+
+      return !isZeroFoundEnd || cursor !== null;
+    }
```

Comment:
Why did you combine this 2 cases if they don't have anything in common?

Reply:
Replaced with 2 separate functions